index.html

```diff
@@ -462,7 +459,9 @@ <h4>Undergraduate</h4>
 					<li>CS 161:
 						<strong>Computer Security</strong>
 						-
-						<a class="current" href="https://www.cs161.org/">s20</a>,
+						<a class="current" href="https://inst.eecs.berkeley.edu/~cs161/sp21/">s21</a>,
+						<a href="https://inst.eecs.berkeley.edu/~cs161/fa20/">f20</a>,
+						<a href="https://www.cs161.org/">s20</a>,
```

Comment:
This didn't resolve for me, but `https://cs161.org/` (without the `www`) works correctly.